sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/TransformWithStateExec.scala

```diff
@@ -92,6 +93,35 @@ case class TransformWithStateExec(
     }
   }
 
+  /**
+   * We initialize this processor handle in the driver to run the init function
+   * and fetch the schemas of the state variables initialized in this processor.
+   * @return a new instance of the driver processor handle
+   */
+  private def getDriverProcessorHandle: DriverStatefulProcessorHandleImpl = {
```

Comment:
nit: let's use `()` explicitly if it's not just a property of class but doing some more thing